whisperkit/pipelines.py

```diff
@@ -102,9 +103,9 @@ class WhisperKit(WhisperPipeline):
     """ Pipeline to clone, build and run the CLI from
     https://github.com/argmaxinc/WhisperKit
     """
-    _compute_unit: CoreMLSwiftComputeUnit = CoreMLSwiftComputeUnit.ANE
-    _randomize_dispatch: bool = False
     _word_timestamps: bool = False
+    _text_decoder_compute_engine = "cpuAndNeuralEngine"
+    _audio_encoder_compute_engine = "cpuAndGPU"
```

Comment:
Could you explain why is the compute engine different for the encoder and the decoder ?

Reply:
There is no reason,  just a default. These could be overridden with either value for either model. This makes me think I should document the intent better 👍

Reply:
Done, also simplified the WhisperKit model downloads!